internal/kafka/internal/workers/kafka_mgrs/ready_kafkas_mgr.go

```diff
@@ -24,15 +25,13 @@ type ReadyKafkaManager struct {
 	keycloakConfig  *keycloak.KeycloakConfig
 }
 
-// NewReadyKafkaManager creates a new kafka manager
-func NewReadyKafkaManager(kafkaService services.KafkaService, keycloakService sso.KafkaKeycloakService, keycloakConfig *keycloak.KeycloakConfig, bus signalbus.SignalBus) *ReadyKafkaManager {
+// NewReadyKafkaManager creates a new kafka manager to reconcile accepted kafkas
+func NewReadyKafkaManager(kafkaService services.KafkaService, keycloakService sso.KafkaKeycloakService, keycloakConfig *keycloak.KeycloakConfig, bus signalbus.SignalBus, reconciler workers.Reconciler) *ReadyKafkaManager {
```

Comment:
```suggestion
// NewReadyKafkaManager creates a new kafka manager to reconcile ready kafkas
func NewReadyKafkaManager(kafkaService services.KafkaService, keycloakService sso.KafkaKeycloakService, keycloakConfig *keycloak.KeycloakConfig, reconciler workers.Reconciler) *ReadyKafkaManager {
```
apply proper commenting and removing unused parameter